### ISSUE TITLE
Breaking Seed Analyzer doesn't drop it's contents

### ISSUE THREAD

**Issue by erima** [InfinityRaider/AgriCraft#57]
After ~15 hours of play seed mutations have vanished from the right-hand pages in the Agricultural Journal. I've tried adding and removing the few mods that were added afterward, to no affect (going so far as to scan more seeds of different types to see if it would repopulate).

**Comment by InfinityRaider** [comment]
My apologies, I don't really understand what's going on? is it just a graphical bug, or are the mutations effectively gone (plants no longer mutate etc.). Can you add some screenshots? And what exactly do you mean with removing and adding other mods?

**Comment by erima** [comment]
I think I figured out the issue. Last night I moved the analyzer with the book still in its slot. Breaking the analyzer with the book in place doesn't drop the book. Apparently the book contains the research (as opposed to Thaumcraft where research is stored in player data), effectively resetting the discovered seed mutations. When I did this I went into creative mode to get a new book, perhaps that is the issue. 

The "missing info" I think, is then just the info being reset, it's likely I didn't get any new mutations after that happened. I thought it might be related to mod conflicts from mods I added to my installation this morning. 

When I just checked on mutations I ended up breaking the analyzer again and the same thing happened. I cheated in another book. By that time I had successfully mutated sugar cane seeds and they appeared. 

In short: mutations don't disappear from books, breaking the analyzer loses book and resets research.

**Comment by InfinityRaider** [comment]
Hmm, seed analyzers used to drop the book, I think that bug appeared after refactoring some tile entity code. I'll fix it, thanks for the report and sorry for your lost data :s
